app/presenters/notification_presenter.rb

```diff
@@ -0,0 +1,90 @@
+class NotificationPresenter < BasePresenter
+  presents :notification
+
+  def avatar_with_link(size)
+    h.link_to(avatar_image(size), 'javascript:void(0)')
+  end
+
+  def created_at_ago
+    h.local_time_ago(notification.created_at)
+  end
+
+  def icon
+    icon_css = %w{notification-icon fa fa-fw}
+    icon_css << case notification.notifiable_type
+                when 'Evidence'
+                  'fa-flag'
+                when 'Issue'
+                  'fa-bug'
+                when 'Node'
+                  'fa-folder-o'
+                when 'Note'
+                  'fa-file-text-o'
+                else
+                  ''
```

Comment:
You need to add a case for `'Comment'` - `fa-comment` is [already being used](https://github.com/dradis/dradis-ce/pull/306/files#diff-fdf1910eed85fd4ade99060ceb5397d9R15) by the activity presenter.

But I don't like how this complicated case statement is duplicated between the two presenters - maybe we could extract a shared method called `fa_icon_for_record` so e.g. `fa_icon_for_record('Evidence')` returns `fa-flag`? Then the code here would be replaced with `icon_css << fa_icon_for_record(notification.notifiable_type)`, with something similar in ActivityPresenter